RecentActivity/src/org/sleuthkit/autopsy/recentactivity/RAImageIngestModule.java

```diff
@@ -87,6 +88,7 @@ public void startUp(IngestJobContext context) throws IngestModuleException {
         extractors.add(registry); // this should run after quicker modules like the browser modules and needs to run before the DataSourceUsageAnalyzer
         extractors.add(osExtract); // this needs to run before the DataSourceUsageAnalyzer
         extractors.add(dataSourceAnalyzer); //this needs to run after ExtractRegistry and ExtractOs
+        extractors.add(zoneInfo); // this needs to run after the web browser modules
```

Comment:
See comment above about reducing noise.  Similarly, here, I would use addAll(Arrays.asList(.....))  and inline the local variables.

Reply:
I think I will leave this one as is.  Having the individual calls, especially with the comments, makes it clear that there is reason to the ordering of the adds.